### ISSUE TITLE
Pressing Enter inside the Console inserts newline instead of executing code

### ISSUE THREAD

**Issue by dlee** [unbug/MIHTool#2]
Pressing Enter inside the Console inserts newline instead of executing code. This only happens on Chrome (Safari correctly executes code instead of inserting newline).

**Comment by unbug** [comment]
Thanks @dlee , I'll look into that. Im sure you can click the button to run.

**Comment by dlee** [comment]
Which button are you talking about? I don't see a button that lets me execute a console command.

**Comment by unbug** [comment]
you have to success to connect to your target, see the "Remote" section.

**Comment by dlee** [comment]
I'm able to connect and enter the Console. It's just that can't execute any commands in the console:

<img width="433" alt="screen shot 2018-07-07 at 1 40 50 pm" src="https://user-images.githubusercontent.com/7035/42414526-7e632ca4-81eb-11e8-9613-37a0f52a9d86.png">

**Comment by unbug** [comment]
Can you show your "Romote" panel? Is there any target succeed to connect?

**Comment by dlee** [comment]
Yes, I'm connected and can view the Elements. I also see logs from the console if my code outputs them via `console.log()`. I'm just unable to execute new commands inside the Console panel.

<img width="422" alt="screen shot 2018-07-09 at 2 13 44 am" src="https://user-images.githubusercontent.com/7035/42441740-c290d326-831d-11e8-89ad-7d04db658146.png">

**Comment by unbug** [comment]
The is weird, can you try in other pages?

**Comment by Grahnz** [comment]
> Pressing Enter inside the Console inserts newline instead of executing code. This only happens on Chrome (Safari correctly executes code instead of inserting newline).

+1, I have the same issue

**Comment by unbug** [comment]
Hi @Grahnz , can you see the targets got enable (with green color) in the remote panel? The issue might cause by remote target lose connect.

**Comment by zzpzzh** [comment]
[]()
